src/operations.cc

```diff
@@ -355,4 +355,11 @@ namespace sharp {
       return image.linear(a, b);
     }
   }
+
+  /*
+   * Median filter with mask size m
+   */
+  VImage Median(VImage image, int const m) {
```

Comment:
here, do we still need this?

Reply:
Good spot, yes, this code is no longer used. Please go ahead and remove it and its definition in the header file, thank you.